src/libraries/System.ComponentModel.Annotations/tests/System/ComponentModel/DataAnnotations/ValidatorTests.cs

```diff
@@ -365,6 +365,7 @@ public void ValidateObject_IValidatableObject_Null()
 
             Validator.ValidateObject(instance, context);
         }
+
```

Comment:
Can this be reverted? It isn't necessary and will only cause unnecessary churn in source history.

Reply:
What happened to the new tests that you added previously? I think fixing this bug should introduce some new tests that ensure the bug is fixed.

Reply:
> Can this be reverted? It isn't necessary and will only cause unnecessary churn in source history.

Do you mean revert the removal of the tests?

Reply:
> What happened to the new tests that you added previously? I think fixing this bug should introduce some new tests that ensure the bug is fixed.

I'll adjust the tests to include the new behavior.

Reply:
> Do you mean revert the removal of the tests?

At first I meant "Can this unnecessary blank line be reverted?". Adding unnecessary blank lines is noise in a PR.

But then I realized that we are no longer adding unit tests in this PR. We should always try to add unit tests for bug fixes, to ensure the behavior is correct and doesn't regress in the future.

> I'll adjust the tests to include the new behavior.

Thank you.